Indicators/Stochastic_Oscillator.py

```diff
@@ -1,20 +1,26 @@
 
 import pandas as pd
-data=pd.read_excel('Stochastic-Oscillator.xlsx')
-print(data)
-
-def add_stochastic_oscillator(df, periods=5):
-    copy = df.copy()
-    
-    high_roll = copy["High"].rolling(periods).max()
-    low_roll = copy["Low"].rolling(periods).min()
-    
+
+class stochastic_oscillator:
```

Comment:
```suggestion
class StochasticOscillator:
```

pep8 uyumunu sağlamak için sınıf isimlerinde CamelCase kullanalım.

Reply:
Değişiklik yapıldı.